### PR TITLE
Translate wheel/pinch event from Deflect to middle-mouse zoom to use pixel delta for movement

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: eq/dc/eventHandler.cpp [hunk 1]
```diff
@@ -160,18 +160,19 @@ void EventHandler::_processEvents( const Proxy* proxy )
...
-            if( dcEvent.dy != 0 )
-                event.pointerWheel.yAxis = dcEvent.dy > 0 ? 1 : -1;
-            _computePointerDelta( event );
+            event.type = Event::CHANNEL_POINTER_MOTION;
+            event.pointerMotion.x = x;
+            event.pointerMotion.y = y;
+
+            event.pointerButtonPress.buttons |= PTR_BUTTON2;
+
+            event.pointerMotion.button = event.pointerMotion.buttons;
```
Comment (eile) [inline eq/dc/eventHandler.cpp]: should be ```= PTR_BUTTON2;```. buttons are all pressed (incl. from previous events), button is the one causing the current event.

**Comment by eile** [comment]
Why is the wheel event not working? Seems semantically more correct, and should cause a zoom as well if set up properly.

**Comment by tribal-tec** [comment]
The wheel applies a constant value for each wheel event, but the deflect translates a pinch to wheel as well and there is one wheel event for each pixel change in the pinch event. The pinch on the other hand reports the 'travelled' pixels, and that's what I'm using now.

File: eq/dc/eventHandler.cpp (hunk 1 above)
Comment (tribal-tec) [inline eq/dc/eventHandler.cpp]: True

**Pull request merged**